manpages/lumina-mediaplayer.8

```diff
@@ -1,26 +1,44 @@
-.Dd October 20, 2017
+.Dd October 23, 2017
 .Dt LUMINA-MEDIAPLAYER 8
 .Os 1.3.3
 
 .Sh NAME
 .Nm lumina-mediaplayer
-.Nd is a simple graphical utility that plays various media files. It also has hooks to stream online radio services.
+.Nd is a simple graphical utility that plays various media files. It also has hooks
+to stream online radio services.
```

Comment:
Let's shorten the description: graphical utility to play media files and stream online radio services.